sklearn/ensemble/tests/test_iforest.py

```diff
@@ -233,16 +233,16 @@ def test_iforest_average_path_length():
     # It tests non-regression for #8549 which used the wrong formula
     # for average path length, strictly for the integer case
     # Updated to check average path length when input is <= 2 (issue #11839)
-    result_one = 2.0 * (np.log(4.0) + np.euler_gamma) - 2.0 * 4.0 / 5.0
-    result_two = 2.0 * (np.log(998.0) + np.euler_gamma) - 2.0 * 998.0 / 999.0
+    result_5 = 77.0/30.0
+    result_999 = 12.9689417211006898253130364
```

Comment:
where is this number coming from?

Reply:
These are the exact values for the test cases, can be calculated using e.g. SymPy.

Reply:
so this is a number with a finite number of decimals?

Reply:
No, it's a rounded rational number. It can be written as a fraction, but then the numerator as well as the denominator are both several hundred digits long.

Reply:
Then maybe replace the exact value comment by saying that it is a rounded rational number

Reply:
ok, fixed